genesis/src/main.rs

```diff
@@ -240,7 +252,7 @@ fn features_to_deactivate_for_cluster(
 fn add_validator_accounts(
     genesis_config: &mut GenesisConfig,
     pubkeys_iter: &mut Iter<Pubkey>,
-    bls_pubkeys_iter: &mut Iter<BLSPubkey>,
+    bls_pubkeys_iter: &mut Iter<BLSPubkeyCompressed>,
```

Comment:
Passing in the `&mut Iter<_>` isn't great. I know your initial PR matched convention and this PR is only tweaking; this is not a blocker for merging this PR but calling it out as something we should fix after (ie how we're passing in a single `pubkeys_iter` where the index % 3 determines whether it is identity / vote / stake)